test/runtime/kafka.go

```diff
@@ -114,19 +117,23 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		Expect(endPoints[helpers.Enabled]).To(Equal(1))
 		Expect(endPoints[helpers.Disabled]).To(Equal(2))
 
+		By("Creating kafka topics")
 		createTopic(allowedTopic)
 		createTopic(disallowTopic)
 
+		By("Listing created Kafka topics")
 		res := vm.ContainerExec("client",
 			"/opt/kafka/bin/kafka-topics.sh --list --zookeeper zook:2181")
 		res.ExpectSuccess("Cannot get kafka topics")
 
+		// Waiting for kafka broker to be up.
+		helpers.WaitKafkaBroker("client", produceCmd, vm)
```

Comment:
Shouldn't this be done after spinning up the broker?

Reply:
I spoke with Romain, and `list topics` and `create topics` communicates directly with zookeeper, so my initial understanding that if list topics succeeds, Kafka broker is up, was wrong. 
Hence I ended up using `Produce to topic instead` as a way to test broker.  This happens before we consume/produce for tests...

Since topics are created afterwards, I put this check here. ? Please let me know if you feel thats not the right approach

Reply:
OK, that makes sense.